src/main/steps/applicant1/hub-page/sole/content.ts

```diff
@@ -30,7 +30,7 @@ const en = ({ isDivorce, partner, formState }: CommonContent) => ({
     } by ${
       formState?.dueDate || dayjs().add(17, 'day').format('D MMMM YYYY')
     }. They can still respond and have been sent a reminder. You can also contact them to remind them if it’s safe to do so.`,
-    line2: `If you do not think they will respond then you can <a class="govuk-link" href="">view the options for proceeding with your ${
+    line2: `If you do not think they will respond then you can <a class="govuk-link" href="/how-you-can-proceed">view the options for proceeding with your ${
```

Comment:
best to have the variable from urls files.
```suggestion
    line2: `If you do not think they will respond then you can <a class="govuk-link" href="${HOW_YOU_CAN_PROCEED}">view the options for proceeding with your ${
```